README.md

```diff
@@ -91,7 +91,7 @@ To run Gradualizer from rebar3, add it as a plugin in your `rebar.config`:
 
 ```erlang
 {plugins, [
-  {gradualizer, {git, "git@github.com:josefs/Gradualizer.git", {branch, "master"}}}
+  {gradualizer, {git, "https://github.com/josefs/Gradualizer.git", {branch, "master"}}}
```

Comment:
Why these changes?

Reply:
`git://` scheme was dropped completely by GitHub, so I wanted to update the docs. On the occasion it doesn't hurt to unify all the examples to use the same scheme.

Reply:
`git@github.com` was not dropped. Only git:// (without username) was dropped IUUC.